sources/packages/backend/libs/integrations/src/services/disbursement-schedule-errors/disbursement-schedule-errors.service.ts

```diff
@@ -61,12 +73,40 @@ export class DisbursementScheduleErrorsService extends RecordDataModelService<Di
       disbursementFeedbackError.creator = auditUser;
       return disbursementFeedbackError;
     });
-    return this.repo
-      .createQueryBuilder()
-      .insert()
-      .into(DisbursementFeedbackErrors)
-      .values(disbursementFeedbackErrors)
-      .orIgnore()
-      .execute();
+    const user =
+      disbursementSchedule.studentAssessment.application.student.user;
```

Comment:
This would be a system action IMO and we can use the `auditUser`. Does it make sense?

Reply:
Yes, the system user.